docs/conops.md

```diff
@@ -2,13 +2,13 @@
 
 <center>
 
-<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
```

Comment:
This seems to be the old style of adding the banner. We don't use <center/> tags anymore. This should be:
```
![Aetheric Banner](https://github.com/aetheric-oss/.github/blob/main/assets/doc-banner.png)
```

Reply:
https://github.com/aetheric-oss/svc-contact/pull/10/commits/716a5407b4bf47dd6fd513153b56a69d3c6da709